GSA-TTS/FAC
# Squash django migrations and remove django-fsm references

Closes #4296.

This PR squashes the existing migrations (which removes the reference to django-fsm's `FSMField`) and removes django-fsm as a dependency.

**IMPORTANT**
- [ ] This PR should NOT be merged into `main` until the deployment on 10/2/2024. This is to prevent any potential conflict with the new migration file that is pending deployment.
- [ ] After this PR is deployed to production (next week ~ 10/9/2024), a new PR can be raised to delete the old migration files.

## How to test

Run a `make docker-clean` and then `make docker-first-run`. The migrations should succeed with no error. Additionally, some basic e2e tests are nice just to sanity check the changes.

## PR Checklist: Submitter


## PR Checklist: Reviewer


The larger the PR, the stricter we should be about these points.

## Pre Merge Checklist: Merger
